collector/lib/NetworkStatusNotifier.cpp

```diff
@@ -305,7 +305,16 @@ void NetworkStatusNotifier::AddConnections(::google::protobuf::RepeatedPtrField<
   int64_t per_container_limit = config_.PerContainerRateLimit();
   CountLimiter rate_limiter(per_container_limit);
 
+  CLOG(DEBUG) << "Per container rate limit: " << per_container_limit;
```

Comment:
Do we need to log this every time we scrape? Would it make sense to print it when it is updated instead?

Reply:
My thinking was to keep the log message close to any rate limiting messages, and to the total vs. sent message. I can probably incorporate it into either of those, though, instead of having this extra message.